### DISCUSSION TITLE
Terra classic: Failed to broadcast when enabling auto-compound

### DISCUSSION THREAD

**Discussion by gmt-it** [classic-terra/core#527] [Support]
Hi.

I've been using RESTAKE for a while but there are problems revoking a grant. The programmers at Restake have opened an issue on terra.js but haven't received a response since July.
[Issue at terra.js](https://github.com/terra-money/terra.js/issues/367)

Here is the issue on restake:
[Issue at restake](https://github.com/eco-stake/restake-ui/issues/89)

Is there no other way for Lunc to report bugs than these discussions?

Best regards

Guido

**Comment by StrathCole** [comment]
Afaik terra.js is not maintained since a long while now. I would assume that after the TFL wind down it won't be maintained either as it has a successor (feather.js) which sadly doesn't implement everything.
